apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/MetadataFixtureBase.cs

```diff
@@ -64,7 +68,8 @@ protected MetadataFixtureBase()
             int port = ((IPEndPoint)temp.LocalEndpoint).Port;
             temp.Stop();
 
-            Environment.SetEnvironmentVariable(EmulatorEnvironmentVariable, $"localhost:{port}");
+            emulatorHost = $"localhost:{port}";
+            Environment.SetEnvironmentVariable(EmulatorEnvironmentVariable, emulatorHost);
```

Comment:
Yes, but this change has no effect other than to cache the environment variable value in a field as well so we don't need to query the variable later.